### PR TITLE
Updated crontab readme

### PR DESCRIPTION
**Pull request by willemdh**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -124,17 +124,23 @@ If you don't have root access, just install everything on your user's folder and
 
 This is an example on how to record the system update package count daily.  
 This will update the file `/var/tmp/updatecount.txt` for later access.  
-Root privilege is required for this operation.
+Root privilege is required for this operation. 
+Only /etc/crontab and the files in /etc/cron.d/ have a username field.
```
Comment (ThomDietrich) [inline readme.md]: Better put these in `code brackets`?

File: readme.md [hunk 2]
```diff
@@ -124,17 +124,23 @@ If you don't have root access, just install everything on your user's folder and
...
+ 
+##### So there you can do this:
```
Comment (ThomDietrich) [inline readme.md]: why as a headline?

File: readme.md [hunk 3]
```diff
@@ -124,17 +124,23 @@ If you don't have root access, just install everything on your user's folder and
...
 
-##### To edit root's crontab
 ```bash
-sudo crontab -e
+sudo vim /etc/crontab
```
Comment (ThomDietrich) [inline readme.md]: Honestly I'd not recommend editing the crontab directly. On my system these lines are inside `/etc/cron.d/firemotd`

File: readme.md [hunk 4]
```diff
@@ -124,17 +124,23 @@ If you don't have root access, just install everything on your user's folder and
...
+# FireMotD system updates check (randomly execute between 0:00:00 and 5:59:59)
+0 0 * * * root perl -e 'sleep int(rand(21600))' && /usr/local/bin/FireMotD -S &>/dev/null
 ```
 
-##### Then add this line
+##### From root's crontab you can use:
```
Comment (ThomDietrich) [inline readme.md]: Again my personal opinion: I would prefer the above solution and leave this out completely. I barely see it used and this method is less clean compared to plain files below `/etc`. The command `crontab -e` is a good solution for non-sudo system users, which firemotd will not be useful for.

File: readme.md (hunk 3 above)
Comment (willemdh) [inline readme.md]: Thanks @ThomDietrich `/etc/cron.d/firemotd` might be the better option indeed. But a lot of users ad systems I now still use `sudo crontab -e` and put in the user field, as per your example. Got some complaints it isn't working in those cases.

**Pull request closed without merging**